src/platform/zcu102_revmin/zcu102_revmin_pfm.tcl

```diff
@@ -0,0 +1 @@
+zcu104_pfm.tcl
```

Comment:
Please do not use link for this file. At lease the description of the platform is different.

Reply:
Hi Min,
I will change it to be a file instead of softlink